unstract/task-abstraction/tests/contract/test_celery_adapter_contract.py

```
@@ -0,0 +1,472 @@
+
+        await celery_client.shutdown()
+
+    @pytest.mark.asyncio
+    async def test_celery_task_inspect(
+        self,
+        celery_client: CeleryTaskClient,
+        sample_workflow_definition: WorkflowDefinition,
+    ):
+        """Test Celery task inspection capabilities."""
+        await celery_client.register_workflow(sample_workflow_definition)
+        workflow_id = await celery_client.run_workflow_async(
```

Ruff (0.13.1)

Local variable `workflow_id` is assigned to but never used

Remove assignment to unused variable `workflow_id`

(F841)